app/src/main/java/com/kickstarter/ui/adapters/ShippingRulesAdapter.kt

```diff
@@ -42,8 +45,8 @@ class ShippingRulesAdapter(ctx: Context, private val resourceId: Int, val items:
         }
     }
 
-    override fun viewHolder(layout: Int, view: View): KSArrayViewHolder {
```

Comment:
Something off on the Shipping Rule Selector.
Attached video comparing it with the latest master

https://user-images.githubusercontent.com/4083656/128768361-d3b440f6-376d-4915-b1ba-6d63013d1442.mp4